src/com/oltpbenchmark/benchmarks/auctionmark/AuctionMarkWorker.java

```diff
@@ -87,7 +93,9 @@ public void run() {
             Procedure proc = AuctionMarkWorker.this.getProcedure(txnType);
             assert(proc != null);
             
-            long sleepTime = AuctionMarkConstants.CLOSE_AUCTIONS_INTERVAL / AuctionMarkConstants.TIME_SCALE_FACTOR;
+            long sleepTime = AuctionMarkConstants.CLOSE_AUCTIONS_INTERVAL / AuctionMarkConstants.TIME_SCALE_FACTOR + 
```

Comment:
Sleep for 20 seconds with a 10 second jitter. We don't want everyone to start at the same time, because if they do then they may not have any auctions to close, which screws with parameter mappings.